docs/team/ngiaphin.md

```diff
@@ -1,7 +1,57 @@
-# Ngiap Hin - Project Portfolio Page
+# Koh Ngiap Hin - Project Portfolio Page
```

Comment:
probably have to rename the file to your github username

Reply:
Updated file name